File: mat/size_ok.m
function [ok] = size_ok()
ok      = true;
x       = [ 1 2 3 ; 4 5 6 ];
% ----------------
[m, n]  = size(x);
% ----------------
ok      = ok & (m == 2);
ok      = ok & (n == 3);
return

File: cpp/size_ok.cpp
# include <mat2cpp.hpp>
bool size_ok(void)
{	bool   ok  = true;
	using namespace mat2cpp;

	size_t m(2), n(3);
	matrix<double> x(m, n);
	// --------------------
	ok &= (x.size1() == m);
	ok &= (x.size2() == n);
	// --------------------
	return ok;
}
